apiary.apib

```diff
@@ -106,4 +106,41 @@ Update referral system for specific organization
         
 + Response 403 (application/json)        
 
-        {status:0 , msg:"Only organization administrators can update referrals."}           
+        {status:0 , msg:"Only organization administrators can update referrals."}        
+        
+### Get referral system subscribers [/organizations/1/referral-system/] [GET]        
+
++ Response 200 (application/json)
```

Comment:
@hakob9315 here we have `subscriber_date`, `created_at`, `updated_at`. I need a date of registration and date of subscription for 'confirmed' referrals and date of registration only for 'potential' referrals. Please describe what is what.
Also, this response should contain `user_name` and `is_new`.
`is_new` - boolean. It should let me know if user already seen this referral. So, if specific referral was not included in this response yet then user doesn't know about him and this referral should have `is_new`: `true`.

Reply:
created_at  date of registration
subscriber_date  date of subscription